addons/website/controllers/main.py

```diff
@@ -1052,7 +1052,9 @@ def update_footer_template(self, template_key, possible_values):
             classes = res[0].get('class').split()
             width = next((c for c in container_classes if c in classes), False)
             if width:
-                views_enable += [width_views.get(width)]
+                view = width_views.get(width)
+                if view is not None:
+                    views_enable += [view]
```

Comment:
- The commit message here still not uses the 72 characters per line, you break at like 60 :shrug: Please configure your setup to properly use those 72 characters.
- More importantly: since your commit mentions this is broken since 18.3, this PR should now targets 18.3. So:
    - Mark the PR as draft (I already did it)
    - Change the PR target to 18.3 (you have to make the PR draft first to not risk pinging people with the retarget)
    - Rebase your branch on 18.3, then push-force here (keeping the same branch name)
    - Re-check everything is fine, then mark your PR as ready for review and ping me again